assets/facets.js

```diff
@@ -154,11 +160,31 @@ class FacetFiltersForm extends HTMLElement {
     ]
   }
 
+  createSearchParams(form) {
+    const formData = new FormData(form);
+    return new URLSearchParams(formData).toString();
+  }
+
+  onSubmitForm(searchParams, event) {
+    FacetFiltersForm.renderPage(searchParams, event);
+  }
+
   onSubmitHandler(event) {
     event.preventDefault();
-    const formData = new FormData(event.target.closest('form'));
-    const searchParams = new URLSearchParams(formData).toString();
-    FacetFiltersForm.renderPage(searchParams, event);
+    const sortFilterForms = document.querySelectorAll('facet-filters-form form');
+    if (event.srcElement.className == 'mobile-facets__checkbox') {
+      const searchParams = this.createSearchParams(event.target.closest('form'))
+      this.onSubmitForm(searchParams, event)
+    } else {
+      const forms = [];
+      sortFilterForms.forEach((form) => {
+        if (form.id == 'FacetSortForm' || form.id == 'FacetFiltersForm') {
```

Comment:
This could even be a 1-liner

```
if (form.id === 'FacetSortForm' || form.id === 'FacetFiltersForm') forms.push(this.createSearchParams(form))
```